src/pages/Like/LikeProduct.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import './LikeProduct.scss';
+
+function LikeProduct({
+  id,
+  img,
+  name,
+  category,
+  price,
+  handleSingleChecked,
+  checkedList,
+}) {
+  return (
+    <tr className="like-product-content" id={id}>
+      <td className="product-content-input">
+        <input
+          type="checkbox"
+          id={id}
+          checked={checkedList.includes(id)}
```

Comment:
위 리뷰대로라면, 아래와 같은 props로 관리할 수 있을 것 같습니다.
```suggestion
          checked={isChecked)}
```